### PR TITLE
Fixed Compiling Raylib for android Using cmake

### PR DESCRIPTION
**Pull request by hero2002**

### CODE REVIEW

File: cmake/LibraryConfigurations.cmake [hunk 1]
```diff
@@ -40,10 +40,18 @@ elseif (${PLATFORM} MATCHES "Android")
     set(PLATFORM_CPP "PLATFORM_ANDROID")
     set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-    
+    set(raylib_sources
```
Comment (raysan5) [inline cmake/LibraryConfigurations.cmake:43]: It seems `raudio.c` is missing...

Comment (hero2002) [reply cmake/LibraryConfigurations.cmake:43]: > It seems `raudio.c` is missing...
yes it's missing but it's missing in the main cmake where I took so It's not from my side

Comment (hero2002) [reply cmake/LibraryConfigurations.cmake:43]: so I will push another pull to fix this and the main one

Comment (raysan5) [reply cmake/LibraryConfigurations.cmake:43]: I think it should be added and tested.

Comment (hero2002) [reply cmake/LibraryConfigurations.cmake:43]: I will add it as option too

Comment (hero2002) [reply cmake/LibraryConfigurations.cmake:43]: wait a minute it's there

Comment (hero2002) [reply cmake/LibraryConfigurations.cmake:43]: in src/CMakeLists.txt line 45

**Comment by raysan5** [comment]
@hero2002 thanks for the improvement

**Pull request merged**